### PR TITLE
Added Demo Page And Few Button Changes

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**

### CODE REVIEW

File: app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -119,16 +141,20 @@ public function login(Request $request)
      */
     public function redirectPath()
     {
+        // dd('vbn');
```
Comment (sandesh556) [inline app/Http/Controllers/Auth/LoginController.php:144]: Remove all the commented dd

File: app/Http/Controllers/Common/BaseSettingsController.php [hunk 2]
```diff
@@ -331,4 +331,52 @@ public function captchaDetails(Request $request)
 
         return ['message' => 'success', 'update' => 'Recaptcha Settings Updated'];
     }
+
+      //Save Google recaptch site key and secret in Database
+    public function v3captchaDetails(Request $request)
+    {
+        $status = $request->input('status');
+        if ($status == 1) {
```
Comment (sandesh556) [inline app/Http/Controllers/Common/BaseSettingsController.php:339]: you just use $status

**Review by sandesh556 — CHANGES_REQUESTED** [review]

**Review by sandesh556 — DISMISSED** [review]

**Review by SarfrazS — CHANGES_REQUESTED** [review]
- [ ] User is requesting a demo, Country code showing twice on the company email.
image

- [ ]After the demo request is sent, but In the Helpdesk ticket gets created with the billing system email. How to support team will coordinate with them. The user is not registered in the helpdesk so they can't add in Cc also. The second thing is when the agent replies again will go to the billing system mail.
image

- [ ] For more user friendly please add here Book Demo for Helpdesk/ServiceDesk.
image

- [ ] No alert message showing when we are passing an invalid email format.
image

- [ ] Query:- After logged in why we are not allowing requests for a demo?

**Comment by bhanu2217** [comment]
**Changes suggested for demo page:**

- Give option in the admin panel to enable/disable this page
- Top Menu bar design is breaking, compare it with original/live billing page
- Show me another option for the buttons Signup for free, request a demo. Move Signup for free to right
- Country code on first time opening should be based on users IP address.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 3]
```diff
@@ -81,7 +81,7 @@ public function postRegister(ProfileRequest $request, User $user)
                 'profile_pic' => '',
                 'active' => 0,
                 'mobile_verified' => 0,
-                'country' => $request->input('country'),
+                // 'country' => $request->input('country'),
```
Comment (RafficMohammed) [inline app/Http/Controllers/Auth/RegisterController.php:84]: remove this line

File: app/Http/Controllers/Common/BaseSettingsController.php [hunk 4]
```diff
@@ -356,4 +356,52 @@ public function captchaDetails(Request $request)
...
+        } else {
+            $nocaptcha_sitekey = '00';
+            $captcha_secretCheck = '00';
+            $path_to_file = base_path('.env');
+            $file_contents = file_get_contents($path_to_file);
+            $file_contents_secretchek = str_replace([env('RECAPTCHA_SITE_KEY'), env('RECAPTCHA_SITE_KEY')], [$captcha_secretCheck, $nocaptcha_sitekey], $file_contents);
+            file_put_contents($path_to_file, $file_contents_secretchek);
+        }
+
+        StatusSetting::where('id', 1)->update(['v3recaptcha_status' => $status]);
```
Comment (RafficMohammed) [inline app/Http/Controllers/Common/BaseSettingsController.php:401]: Don't include id directly

**Comment by bhanu2217** [comment]
The email that goes out in that email Reply to email address should be of the customer who filled the form.

**Comment by bhanu2217** [comment]
@sowmiyasubramaniyan mentioned this form is using Faveo's create ticket API.

Agora Invoicing is designed like a product. How will this page work for other's who are using this software and not using Faveo Helpdesk?
Ideally we should have used simple mail functionality with reply to

Awaiting your response.

**Comment by bhanu2217** [comment]
**Feedback:**

**Setting changes:**

- Keep only page enable/disable button
- To email will be system email
- Reply to will be customer filled email

**Review by SarfrazS — CHANGES_REQUESTED** [review]
Last commit
image
I ran php artisan migrate
Unable to test getting below error. Please fix and let me know then i will continue testing.
image

**Comment by sowmiyasubramaniyan** [comment]
New PR : https://github.com/ladybirdweb/agora-invoicing-community/pull/2447

**Pull request closed without merging**